docker-compose.yml

```diff
@@ -2,12 +2,13 @@ version: '2'
 services:
 
   web:
-    image: fpfis/php56-dev
+    # image: fpfis/httpd-php-dev:5.6
+    image: fpfis/httpd-php-dev:7.1
     working_dir: /var/www/html
+    ports:
+      - 8080:8080
     volumes:
       - .:/var/www/html
     environment:
       XDEBUG_CONFIG: "remote_enable=1 remote_host=${DOCKER_HOST_IP} remote_port=9000 idekey=PHPSTORM remote_autostart=1"
       PHP_IDE_CONFIG: "serverName=Test"
-      COMPOSER_MEMORY_LIMIT: "2G"
-      COMPOSER_CACHE_DIR: "/tmp/composer"
```

Comment:
I had doubts about this change. Memory limit is managed in the image and cache should also be (with @gboddin 's approval).